lib/Doctrine/ORM/NativeQuery.php

```diff
@@ -60,7 +60,7 @@ public function getSQL()
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```

Comment:
what was your motivation to change these? I fuzzily remember phpcs understood lowercase variant only

Reply:
I don't remember, and we seem to be using a mix of both in the project :sweat: .

Will look into it later.